.github/workflows/ci.yml

```diff
@@ -76,12 +76,12 @@ jobs:
         python-version: "3.13"
     - run: |
```

Comment:
(related to the commit message of the first commit) The paragraph starting with `Adding `setup.py` back in could be a fall-back solution` is not wrapped in the commit message.